docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+
+Network Perspectives MUST be spread across a minimum of two distinct continents. Each Network Perspective MUST independently complete the entirety of:
+- the domain validation process used to establish the Primary Domain Validation Determination (i.e., including DNS lookups for A records), and
+- the CAA record checking process used to establish the Primary CAA Determination. 
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives).
+
+In the event that the Primary Domain Validation Determination or Primary CAA Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table, CAs MAY immediately retry Multi-Perspective Domain Validation using the same validation method or an alternative method (e.g., a CA can immediately retry validation using “Email to DNS TXT Contact” if “Agreed-Upon Change to Website - ACME” does not corroborate the outcome of Multi-Perspective Domain Validation). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+Phased Implementation Timeline:
+- *Effective March 15, 2024*, the CA SHOULD implement Multi-Perspective Domain Validation as described in this section. 
+- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MAY proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table.
+- *Effective March 15, 2025*, the CA MUST implement Multi-Perspective Domain Validation as described in this section; however, the CA MUST NOT proceed with certificate issuance if the Primary Domain Validation Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table and the "Defining Corroborating CAA Evidence" table.
+
+Even though the "# of Network Perspectives and Quorum Requirements" table permits issuance when some Network Perspectives do not corroborate the Primary Domain Validation Determination, the set of Network Perspectives that do corroborate the Primary Domain Validation Determination MUST be spread over two distinct continents for a CA to proceed with certificate issuance after March 15, 2025.
```

Comment:
I think the language in this section is unclear as to whether it is placing requirements on the _total_ set of network perspectives (where the Primary Domain Validation Determination is made, plus the additional Network Perspectives), or if this section is placing requirements _only_ on the additional Network Perspectives which are attempting to corroborate the Primary Domain Validation Determination.

For example, if my primary (audited) location is in NA, and all of my MPDV additional perspectives are in Europe, does that satisfy the "two continents" requirement? Or do my MPDV additional perspectives themselves need to be in both NA and Europe, regardless of where my primary (audited) location is?

Reply:
That's a great point. My personal preference would be to require the additional MPDV perspectives to be spread across two distinct continents independent of where the original datacenter is. This introduces more spread in the MPDV points and reduces the risk that all of them can be compromised with a single attack.

Reply:
> MUST be spread over two distinct continents for a CA to proceed with certificate issuance after March 15, 2025

This conflicts with (or further constrains) the normative requirements included in the definition of Network Perspectives on L382.

> Related to Multi-Perspective Domain Validation. A configuration for sending outbound Internet traffic associated with a domain control validation method. The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is first handed off to the network infrastructure providing Internet connectivity to that perspective. This infrastructure may be administered by the same organization providing the computational services required to operate the perspective. Network Perspectives are considered distinct when the straight-line distance between the two States or Provinces they reside in are separated by a distance of at least 500 km.

Is this clause even required if by definition network perspectives must be diverse?

Reply:
@aarongable / @birgelee - I'm not sure I understand the suggested edit. Can you please explain further?

I interpret the following "_...the set of Network Perspectives that do corroborate the Primary Domain Validation Determination MUST be spread over two distinct continents for a CA to proceed with certificate issuance after March 15, 2025_."  .... to satisfy that desired intent (i.e., regardless where the primary determination was made, that determination must be corroborated by perspectives spanning at least 2 distinct continents.)


@slghtr-says - I believe the intent of two distinct continents in addition to the 500km separation is to improve resistance to what might be considered "regional" BGP attacks. I'll leave it to @birgelee or another member of the Princeton Team to serve as the authoritative source for this justification.